### PR TITLE
Persist DB by default

### PR DESCRIPTION
**Pull request by lucafavatella**
Delivers https://www.pivotaltracker.com/story/show/165168390

TODO:
* [x] Update docs

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -62,14 +62,8 @@ More details about node configuration can be found in [configuration documentati
...
-```yaml
-# ... SNIP
-chain:
-    persist: true
-# SNIP ...
-```
+The blockchain data is persisted by default, inside the Docker container.
+In order to persist the data in a directory on the host machine, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/).
 
 Assuming your configuration file path is `~/.aeternity/myaeternity.yaml` on host machine.
```
Comment (dincho) [inline docs/docker.md]: This line could be removed as well, also the config mount in the command below.

File: docs/docker.md [hunk 2]
```diff
@@ -62,14 +62,8 @@ More details about node configuration can be found in [configuration documentati
...
 
-To persist blockchain data between container runs use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/) and enable chain persistence in your configuration:
-
-```yaml
-# ... SNIP
-chain:
-    persist: true
-# SNIP ...
-```
+The blockchain data is persisted by default, inside the Docker container.
```
Comment (dincho) [inline docs/docker.md]: It's worth nothing mentioning that by default the database is persisted at: `/home/aeternity/node/data/mnesia` in the container.

Comment (lucafavatella) [reply docs/docker.md]: I expect it benefits Docker image user re which parameters to tune when starting container. User may be surprised to find that the new container fails after a while if container started with large memory but little storage.

File: docs/docker.md (hunk 1 above)
Comment (lucafavatella) [reply docs/docker.md]: You are right. Amending.

Comment (lucafavatella) [reply docs/docker.md]: Addressed in 369af4090

File: docs/docker.md [hunk 3]
```diff
@@ -62,22 +62,14 @@ More details about node configuration can be found in [configuration documentati
 
 ### Persisting Data
 
-To persist blockchain data between container runs use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/) and enable chain persistence in your configuration:
+The blockchain data is persisted by default, inside the Docker container.
```
Comment (dincho) [inline docs/docker.md]:
```suggestion
The blockchain data is persisted by default at `/home/aeternity/node/data/mnesia`, inside the Docker container.
```

File: docs/docker.md (hunk 2 above)
Comment (lucafavatella) [reply docs/docker.md]: (I had not got initial comment, you suggested change now included. 👍 )

**Pull request merged**